paseos/paseos.py

```diff
@@ -95,13 +118,25 @@ def advance_time(self, time_to_advance: float):
             # Perform updates for local actor (e.g. charging)
             # Each actor only updates itself
             # charge from current moment to time after timestep
-            self._local_actor.charge(
-                self._local_actor.local_time,
-                pk.epoch((self._state.time + dt) * pk.SEC2DAY),
-            )
+            if self.local_actor.has_power_model:
+                self.local_actor.charge(
+                    self.local_actor.local_time,
```

Comment:
I'll change it this time, but in general I would suggest to avoid changing things unrelated to PRs in them and instead create issues :v:. Otherwise, we end up with a lot of merge conflicts and wonder why things changed in PRs dedicated to completely different topics.

Reply:
I agree with that. It was more of a question and not a query to change here. :)